starter/src/index.html

```diff
@@ -55,20 +15,16 @@
 	<!-- Webpack Dev Server reload -->
 	<script src="/webpack-dev-server.js"></script>
 	<% } %>
-
+	
+	<!-- Webpack DLL CSS files -->
+	<!-- The base href is defined through the Webpack configuration and can be customized through the angular.json file -->
```

Comment:
I've just noticed that in my comment in the previous review the element I was referring to did not appear :( sorry.

What I meant was this: "These comments should be put before the `<meta charset="utf-8">` element above (on line 4)"

Reply:
Nooooooo... :p I'll do it right now ;)